packages/nodejs-ext/scripts/extension/helpers.js

```diff
@@ -28,4 +28,13 @@ function hasSupportedArchitecture(report) {
   )
 }
 
-module.exports = { hasLocalBuild, hasSupportedArchitecture }
+/**
+ * Returns `true` if the current linux system is using musl as its libc and vice versa
+ *
+ * @return  {boolean}
+ */
+function hasMusl() {
+  return /musl/.test(child_process.spawnSync("ldd", ["--version"]).stderr);
```

Comment:
This currently throws as `child_process` is not imported from anywhere. Can you please add the necessary import to the top of the file?